ttnn/cpp/ttnn/operations/moreh/moreh_softmax/device/moreh_softmax_device_operation.cpp

```diff
@@ -9,7 +9,7 @@ namespace ttnn::operations::moreh::moreh_softmax {
 #define L1_512KB (512 * 1024)
 
 bool is_moreh_softmax_w_small_available(const Tensor& tensor, const DeviceComputeKernelConfig& compute_kernel_config) {
-    auto w = tensor.get_legacy_shape()[-1];
+    auto w = tensor.get_shape().value[-1];
```

Comment:
```suggestion
    auto w = tensor.get_shape()[-1];
```
I think it can be used like this.